source/codegen/metadata/nisync/config.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+config = {
+    "api_version": "20.6.0d7",
```

Comment:
Are you checking in a pre-release version of your C header? I don't have a problem with that, but I want to make sure you're aware that this is getting published to a public open-source repo.

Reply:
Thanks for pointing it out. Updated version to 21.1.0